src/test/kotlin/io/perfometer/data/csv/CsvParserSpecification.kt

```diff
@@ -0,0 +1,67 @@
+package io.perfometer.data.csv
+
+import io.kotest.matchers.shouldBe
+import kotlin.test.Test
+
+internal class CsvParserSpecification {
+
+    @Test
+    fun `should parse unquoted csv`() {
+        // given
+        val csvText = """
+            1,Marcin,Nowak
+            2,Jan,Kowalski
+            3,Bartek,Bąkiewicz
+        """.trimIndent()
```

Comment:
Not necessary `trimIndent()` call, I think. By default the "trim" marker is pipe char "|", and in the String there's none, hence nothing will be trimmed.

Reply:
You mean `trimMargin`. `trimIndent` is removing "common minimal indent" so at this case all the whitespace at the beginning of lines.

Reply:
Right, I've confused the methods :(